routes/projects.rb

```diff
@@ -25,6 +25,12 @@ def self.registered(app)
           halt 404, 'Project not found'
         end
         @clients = Client.filter(id: @project.client_id).all
+        if @clients.nil?
+          MyLog.log.fatal "routes/projects: Project's clients not found"
+          halt 404, "Project's Client not found"
+        end
+        @nodes = NodeResource.filter(project_id: @project.id).all
+        @dbs = DBResource.filter(project_id: @project.id).all
```

Comment:
If you use reciprocal associations in the model, you should be able to just get the client, nodes and dbs from the project object: project.nodes, project.client
See http://sequel.jeremyevans.net/rdoc/files/doc/association_basics_rdoc.html and the kennric/reciprocal_relations branch